server.js

```diff
@@ -19,8 +53,40 @@ app.use(cors())
 app.use(bodyParser.json())
 
 // Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+// app.get('/', (req, res) => {
+//   res.send('Hello world')
+// })
+
+app.get('/', async (req, res) => {
+  const thought = await Thought.find().sort({ createdAt: 'desc' }).limit(20).exec()
+  res.json(thought)
+})
+
+app.post('/', async (req, res) => {
```

Comment:
It's not clear what a POST to '/' will do.  

Either naming the endpoint '/thoughts' or adding a comment to the code would be appropriate here.